projects/container_bench/testing/exec_time.py

```
@@ -0,0 +1,184 @@
+               execution_time (float): Time taken to execute the command in seconds.
+    """
+    monitor_thread.start()
+    start_time = time.perf_counter()
+    process = subprocess.Popen(
+        command_list,
+        stdout=subprocess.PIPE,
+        stderr=subprocess.PIPE,
+        text=True,
+        bufsize=1,
+        universal_newlines=True
+    )
```

Pylint (3.3.7)

[refactor] Consider using 'with' for resource-allocating operations

(R1732)